Update requirements.mdx (#808)
bigcommerce/docs: docs/integrations/apps/guide/requirements.mdx

* Apps must use V3 endpoints in favor of V2 endpoints when feature parity exists.
* Apps must serve all callback URLs over HTTPS.
* Apps in the App Marketplace must be [multi-user enabled](/docs/integrations/apps/guide/users).

> * Apps that process transactions or handle credit card data must pass a PCI Compliance review by BigCommerce's security team and provide both a PCI-DSS Attestation of Compliance and a data flow diagram. New payment gateway app submissions are not currently being accepted for App Marketplace review without pre-approval.

Reviewer: ```suggestion
* Apps that process transactions or handle credit card data must pass a PCI Compliance review by BigCommerce's security team and provide both a PCI-DSS Attestation of Compliance and a data flow diagram. BigCommerce only accepts pre-approved new payment gateway app submissions for App Marketplace review.
```

* Apps that access the Checkout Content scope will also be subject to a security assessment by BigCommerce's security team.
* Apps that modify the checkout experience must use the BigCommerce [Checkout SDK](/docs/storefront/cart-checkout/checkout-sdk).
* Apps that add another marketplace or sales channel to a store should make use of the [Channels Toolkit](/docs/integrations/channels/guide) and follow [Channel App Requirements](/docs/integrations/channels/guide/requirements).
* Apps that create orders in the BigCommerce store need to properly mark accurate order source, payment method, and other order details.

> <Callout type="info">We have paused all new externally built payment integrations for our publicly available BigCommerce Marketplace. We are in the process of updating our available APIs to create a more robust experience and hope to re-open this program in the near future. Please reach out to paymentspod@bigcommerce.com with any questions.</Callout>

Reviewer: ```suggestion
<Callout type="info">We have paused all new externally built payment integrations for our publicly available BigCommerce Marketplace. We are updating our available APIs to create a more robust experience and hope to re-open this program in the near future. Please reach out to paymentspod@bigcommerce.com with any questions.</Callout>
```
